frameworks/Python/django/benchmark_config.json

```diff
@@ -47,6 +49,7 @@
     },
     "postgresql": {
       "setup_file": "setup_pg",
+      "plaintext_url" : "/plaintext",
       "json_url": "/json",
```

Comment:
@mwflickner since plaintext and json don't require any DB access, we shouldn't expect any different results on postgresql vs. mysql. can you remove them from the postgresql test since we already have them elsewhere in the django tests?